lib/isolator/isolate.rb

```diff
@@ -3,10 +3,46 @@
 module Isolator
   # Add .isolate function to build and register adapters
   module Isolate
+    TODO_PATH = ".isolator_todo.yml"
+
     def isolate(id, **options)
       raise "Adapter already registered: #{id}" if Isolator.adapters.key?(id.to_s)
       adapter = AdapterBuilder.call(options)
+      register_todos(id.to_s, adapter) if isolator_todos
       Isolator.adapters[id.to_s] = adapter
     end
+
+    private
+
+    def register_todos(id, adapter)
+      ignored_paths = isolator_todos.fetch(id, [])
+      ignored_paths.each { |path| register_todo(adapter, path) }
+    end
+
+    def register_todo(adapter, path)
+      ignored_files = Dir[Rails.root.join(path)]
+
+      if ignored_files.blank?
+        ignore_line(adapter, path)
+      else
+        ignore_in_files(adapter, ignored_files)
+      end
+    end
+
+    def ignore_line(adapter, line)
+      full_line = Rails.root.join(line).to_s
```

Comment:
We cannot use Rails here, Isolator is not Rails-specific.

Looks like we need a way to specify the path to the configuration somehow, e.g. `Isolator.load_ingore_config "./.isolator_blabla.yml"`.

That also means that we cannot apply ignores within `isolate` method, since it's called when gem is loading.

So, we should configure all the ignores in a separate step (we can access existing adapters through `Isolator.adapters`).

To make it easier to use with Rails we can add a `Railtie` with an initializer to load configuration form `Rails.root.join(TODO_CONFIG_PATH)`.